docs/src/main/sphinx/release/release-405.md

```diff
@@ -2,10 +2,15 @@
 
 ## General
 
+* Add support for setting an optional limit for the `query_max_run_time` session
+  property, which can be set using the `query.max-run-time.hard-limit`
+  configuration property. ({issue}`15115`)
 * Improve performance of queries that compare `date` columns with
   `timestamp(n) with time zone` literals. ({issue}`5798`)
 * Improve performance and resource utilization when inserting into partitioned
   and bucketed tables. ({issue}`14718`)
+* Improve performance when querying a small number of partitions while
+  fault-tolerant execution is enabled. ({issue}`14735`)
```

Comment:
@arhimondr look ok?

Reply:
Sorry, I should've provided an easier to comprehend explanation in the PR.

How about something like

> Ensure Trino does not run into a limit on maximum number of partitions  (>100) that can be inserted with a single query when fault tolerant execution is enabled. Queries that run with fault tolerant execution disabled may still run into this limitation.

Reply:
We want to frame it as user impact, so my thought is that this is a performance improvement for these situations.